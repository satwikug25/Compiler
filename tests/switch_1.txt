n, a, b;
{
    input n;
    output n;
    input a;
    input b;
    a = 10; 
    b = 20;
    output a; 
    output b; 
    SWITCH n {
        CASE 1:{
            a = a + 1; 
            output a;
        }
        CASE 2:{  
            b = b + 1;   
            output b;
        }
    }
    output a;
    output b;
}
1 2 3