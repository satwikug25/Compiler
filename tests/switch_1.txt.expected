1 10 20 11 11 20